theme/js/locales/fr.json

```diff
@@ -128,5 +128,5 @@
   "All coverages": "Toutes les couvertures",
   "Spatial granularity": "Granularité spatiale",
   "Finest level of geographic detail covered by data.": "Niveau de détail géographique le plus fin que peut couvrir les données.",
-  "Ex. 2022 presidential election": "Ex: élections présidentielles 2022"
+  "Ex. 2022 presidential election": ""
```

Comment:
This translation is removed from json files since the sync.

Reply:
Good catch, fixed!